### DISCUSSION TITLE
PLEASE make Spiral Linux available by torrent and anywhere other than Sourceforge

### DISCUSSION THREAD

**Discussion by wjlesaulnier** [SpiralLinux/SpiralLinux-project#35] [General]
For about 18 hours per day SourceForge is unusable. I just tried to download Spiral and it says 9 hours to complete. It takes 2 to 5 minutes for me to download a 2GB ISO from anywhere else. Please offer a mirror anywhere else.

**Comment by geckolinux** [comment]
Hi there, please switch to a different Sourceforge mirror and/or use a multi-thread download manager such as `aria2` . I can saturate my 50mbps connection that way.

**Reply by wjlesaulnier** [reply]
I tried 2 different mirrors, but same speed. Will try aria2. And I LOVE what you did for both OpenSuse and Debian installing!!!!!!!

**Reply by wjlesaulnier** [reply]
Something strange I've noticed is that my geek friends from roughly south of the Ohio River and roughly East of the Mississippi River have this same problem with SourceForge in general. But, geek friends that live outside of this southeast area don't have this problem, or switching mirrors quickly solves a slow download.

**Reply by geckolinux** [reply]
Glad you find GeckoLinux and SpiralLinux useful! Sorry for the trouble with Sourceforge, it honestly sounds more like a temporary network/routing issue in your geographic area. Maybe try a mirror in Europe or Asia?

**Comment by jversusk** [comment]
+1 for torrent downloads. My connection is not reliable enough to download ISOs the regular way - the download could contain errors or even fail at 99% & I'd have to start all over again.

**Reply by geckolinux** [reply]
Hi there, while I sympathize with your situation I don't have the capability of creating torrents. However, by using a download manager you will not experience either of the two problems you anticipate-- errors will be detected and corrected, and the download can be resumed if it fails at any point. Let me know if you need recommendations for a good download manager and what your preferences are (browser based, Windows, Linux GUI, Linux terminal, etc.)

**Reply by jversusk** [reply]
Thanks for the suggestion, but I've used download managers in the past and sometimes had errors in the completed file. And unlike torrents, individual pieces can't be verified and re-downloaded once the download is completed. For that reason, I only use download managers for relatively small files.

That said, I understand if you're unable to provide torrent downloads. Hopefully, I will be able to find it on distrowatch or linuxtracker.

**Comment by IdolIcon** [comment]
+1

I have the same problems with Sourceforge here, in Spain. Sourceforge is one the worst places to download a large file like an ISO, using a downloader or not, changing mirrors or not.

**Reply by marwenius** [reply]
There are unofficial torrents created by FOSS Torrents:
https://fosstorrents.com/distributions/spirallinux/
